src/data/pageGroups/section-1.yml

```diff
@@ -1,7 +1,10 @@
+order: 1
```

Comment:
I think it defaults to alphabetically ordered and I didn't feel like writing the custom sort.